server/src/main/java/com/clarkparsia/pellet/server/model/impl/ServerStateImpl.java

```diff
@@ -48,14 +47,10 @@ protected ServerStateImpl(final Iterable<OntologyState> onts) {
 		return Optional.fromNullable(ontologies.get(ontology));
 	}
 
-	protected OntologyState createOntologyState(final String ontologyPath) throws OWLOntologyCreationException {
-		OWLOntology ont = manager.loadOntologyFromOntologyDocument(new File(ontologyPath));
-		return new OntologyStateImpl(ont);
-	}
-
 	@Override
 	public OntologyState addOntology(final String ontologyPath) throws OWLOntologyCreationException {
-		OntologyState state = createOntologyState(ontologyPath);
```

Comment:
I see so much of this in protege, calls to functions that are only used once. Perhaps they anticipate reusing them, but it makes reading them annoying